src/utils.tsx

```diff
@@ -13,9 +13,9 @@ export function mjmlReactComponentFactory<P>(
   };
 }
 
-export function convertPropsToMjmlAttributes<P>(
-  props: { [K in keyof P]: unknown }
-) {
+export function convertPropsToMjmlAttributes<P>(props: {
+  [K in keyof P]: unknown;
+}) {
```

Comment:
It looks like there are quite a few style changes. Is it possible to revert them or is this purely prettier?